Makefile

```diff
@@ -30,63 +30,39 @@ pull: ## Pull all images
 
 	@ echo "Pulling images..."
 
-	@ make pull-development-images
+	@ make pull-development-image
 	@ make pull-release-images
 
-pull-development-images: ## Pull development images
+pull-development-image: ## Pull development image
 
-	@ echo "Pulling development images..."
+	@ echo "Pulling development image..."
 
-	@ make pull-development-image-debian
-	@ make pull-development-image-fedora
-
-pull-development-image-debian: target := debian
-pull-development-image-fedora: target := fedora
-
-pull-development-image-debian pull-development-image-fedora: _pull-development-image
-
-_pull-development-image:
-
-	@ echo "Pulling [$(target)] development image..."
-
-	docker pull $(docker_development_image_repository):$(docker_image_version)-$(target) || true
-	docker pull $(docker_development_image_repository):latest-$(target) || true
+	docker pull $(docker_development_image_repository):$(docker_image_version) || true
+	docker pull $(docker_development_image_repository):latest || true
 
 pull-release-images: ## Pull release images
 
 	@ echo "Pull release images..."
 
-	@ make pull-release-image-cpp-debian
-	@ make pull-release-image-cpp-fedora
+	@ make pull-release-image-cpp
 
-	@ make pull-release-image-python-debian
-	@ make pull-release-image-python-fedora
+	@ make pull-release-image-python
 
 	@ make pull-release-image-jupyter
 
-pull-release-image-cpp-debian: target := debian
-pull-release-image-cpp-fedora: target := fedora
-
-pull-release-image-cpp-debian pull-release-image-cpp-fedora: _pull-release-image-cpp
-
-_pull-release-image-cpp:
-
-	@ echo "Pull [$(target)] C++ release image..."
-
-	docker pull $(docker_release_image_cpp_repository):$(docker_image_version)-$(target) || true
-	docker pull $(docker_release_image_cpp_repository):latest-$(target) || true
+pull-release-image-cpp:
 
-pull-release-image-python-debian: target := debian
-pull-release-image-python-fedora: target := fedora
+	@ echo "Pull C++ release image..."
 
-pull-release-image-python-debian pull-release-image-python-fedora: _pull-release-image-python
+	docker pull $(docker_release_image_cpp_repository):$(docker_image_version) || true
+	docker pull $(docker_release_image_cpp_repository):latest || true
 
-_pull-release-image-python: _pull-development-image
+pull-release-image-python: pull-development-image
```

Comment:
Nope?

```suggestion
pull-release-image-python:
```

Reply:
Yup yup that's a good find! Thank you for going through this in detail. I have definitely been a bit mindless.

Reply:
I will propagate these changes to the other repos